tests/hypothesis/rename_test.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+import string
+
+import pytest
+from hypothesis import given
+from hypothesis import strategies as st
+
+import narwhals as nw
+from tests.utils import Constructor
+from tests.utils import assert_equal_data
+
+strategy = st.lists(
+    st.text(
+        alphabet=st.characters(
+            codec="utf-8", exclude_characters=['"', ".", "'", "\x00", *string.whitespace]
```

Comment:
Both SQLFrame and duckdb fail with those characters. SQLFrame is due to sqlglot parsing.
I will make a repro and open issues upstream